src/function/table/pragma_detailed_profiling_output.cpp

```diff
@@ -32,15 +30,27 @@ static unique_ptr<FunctionData> PragmaDetailedProfilingOutputBind(ClientContext
 	names.emplace_back("OPERATOR_ID");
 	return_types.push_back(LogicalType::INTEGER);
 
-	names.emplace_back("FUNCTION_ID");
+	names.emplace_back("ANNOTATION");
+	return_types.push_back(LogicalType::VARCHAR);
+
+	names.emplace_back("ID");
 	return_types.push_back(LogicalType::INTEGER);
 
 	names.emplace_back("NAME");
 	return_types.push_back(LogicalType::VARCHAR);
 
-	names.emplace_back("TIME");
+	names.emplace_back("TIME/CyclePerTuple");
```

Comment:
Is it time or is it cycles per tuple? If it depends it is better to have two columns (`TIME` and `CYCLES_PER_TUPLE`)  where either time or cycles_per_tuple is set to NULL when not available.

Reply:
It depends on the BUILD_RDTSC flag. If it is not enabled, the cycle counter measures time.

Reply:
Those should be two separate columns. Mixing two different data points in a single column is asking for trouble.